packages/remark-mdx-remove-imports/package.json

```diff
@@ -42,8 +43,8 @@
     "unist-util-remove": "2.0.0"
   },
   "devDependencies": {
-    "remark-parse": "^8.0.0",
-    "remark-stringify": "^8.0.0",
+    "remark-parse": "^9.0.0",
+    "remark-stringify": "^9.0.1",
```

Comment:
is v9.0.1 specifically needed?
Would the broader `^9.0.0` range work?

Reply:
This particular change was because I pushed 9.0.1 while developing and yarn wasn’t picking up on the publish
This package is dropped so a non-issue: i’ll remove the deprecated packages in a next PR.